### ISSUE TITLE
Items in Fruits and Vegetables section in ItemsRepeater gets clipped in 200% Text Scaling.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#580]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisite: 
- Settings--> Ease of Access-->Display-->Drag the Display slider  to  200%-->Apply
- Screen Resolution: 1280 x 768

1. Launch XAML Controls Gallery.
2. Activate ItemsRepeater from home page and observe the Items in Fruits and Vegetables section under the header "Laying out nested ItemsRepeater".

**Expected behavior**

Information under the header "Laying out nested ItemsRepeater" should be clearly visible in 200% text scaling.

**Actual behavior**

Items in Fruits and Vegetables section in ItemsRepeater gets clipped in 200% Text Scaling.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97392236-7f9fa380-189e-11eb-83bf-7a598116bf46.png)

**Version Info**

Gallery version (found on Settings page): 1.2.13.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/25440755)

**Comment by YuliKl** [comment]
Comment in original issue:
This is an app bug, repeater just lays the items out according to its layout, if you want a layout that wraps you need to change the provided layout, or if you want to scroll the content you need to put the content in a scrolling surface. This is the same as if you put a horizontal stack panel with these items in it with no scrollviewer.
